Cargo.toml

```diff
@@ -3,4 +3,5 @@ members = [
   "libursa",
   "libzmix",
   "libzmix/ps",
+  "libzmix/delg_cred_cdd"
```

Comment:
Why are you using a separate crate? We can use the `features` to handle this.

Reply:
Where should this go? In signatures?

Reply:
Yes

Reply:
Done